DESCRIPTION

```diff
@@ -93,7 +93,8 @@ Imports:
     glue (>= 1.3.2),
     bslib (>= 0.6.0),
     cachem (>= 1.1.0),
-    lifecycle (>= 0.2.0)
+    lifecycle (>= 0.2.0),
+    watcher 
```

Comment:
I would feel better if `watcher` wasn't a hard requirement because of its somewhat non-standard system dependencies for building from source.

We do, unfortunately, have lots of users/customers running older versions of R where binaries won't be available

Reply:
I'm not sure we want to go all the way to this end, but in b3f2948ec0e038d55238d7e0f1f25b76e0c9297f I restored the legacy autoreload behavior and moved `watcher` to suggests, with a warning to install `watcher` for an improved experience.